pkg/controller/cassandradatacenter/statefulset.go

```diff
@@ -24,21 +23,9 @@ const (
 	OperatorConfigVolumeMountPath = "/tmp/operator-config"
 	UserConfigVolumeMountPath     = "/tmp/user-config"
 	UserSecretVolumeMountPath     = "/tmp/user-secret-config"
```

Comment:
This probably should just be `/tmp/user-secret` for consistency.

Reply:
:+1: